src/Flight/Flight.java

```diff
@@ -1,14 +1,15 @@
 package Flight;
 
+import java.time.LocalDateTime;
 import java.util.Date; //If you want to see a class you didn't implement ctrl+left-click
```

Comment:
Usage of Date seems to have been replaced with LocalDateTime, importing Date might not be needed and possibly could be removed.